tests/test_hpack.c

```diff
@@ -802,3 +803,48 @@ static int test_hpack_decode_empty_value(struct aws_allocator *allocator, void *
     aws_http_library_clean_up();
     return AWS_OP_SUCCESS;
 }
+
+/* Test header with empty name and value */
+AWS_TEST_CASE(hpack_decode_empty_header, test_hpack_decode_empty_header)
```

Comment:
[style/opinion] A test like this would be much shorter and easier to understand if it used the underlying aws_hpack_insert_header()/aws_hpack_find_index() functions, rather than feeding a binary buffer through the full decode sequence.